### DISCUSSION TITLE
How to use the StackedTabsScaffold

### DISCUSSION THREAD

**Discussion by lobonz** [FilledStacks/stacked#1144] [Q&A]
I am using Stacked in my app and mostly all working well however struggling to get the StackedTabsScaffold to work.
I was following this tutorial:
https://isaacadariku.medium.com/flutter-persistent-navigation-bar-with-nested-navigation-using-stacked-router-service-b41a8b2a33b6

The issue I have is that StackedTabsScaffold's routes property expects PageRouteInfo which in the tutorial appears to use routes that are generated (using stacked generate) however those routes dont seem to be generated for me.

I know Stacked has been through quite a bit of development since that tutorial so expecting this may have changed however there is very little I can find by way of documentation or examples and spent half a day to try and get it working.

If someone has an example or can tell me how to supply the correct routes to StackedTabsScaffold that would help me greatly.

**Comment by RaymondAtivie** [comment]
@lobonz  Did you figure this out? In the same boat

**Reply by lobonz** [reply]
No sorry, gave up.
Ended up rolling my own. I cant remember exactly how I did it but can check it out when I'm back at work if you are interested what I came up with.
